coderd/templateversions_test.go

```diff
@@ -758,45 +886,49 @@ func TestPaginatedTemplateVersions(t *testing.T) {
 	}{
 		{
 			name: "Single result",
-			args: args{ctx: ctx, pagination: codersdk.Pagination{Limit: 1}},
+			args: args{pagination: codersdk.Pagination{Limit: 1}},
 			want: templateVersions[:1],
 		},
 		{
 			name: "Single result, second page",
-			args: args{ctx: ctx, pagination: codersdk.Pagination{Limit: 1, Offset: 1}},
+			args: args{pagination: codersdk.Pagination{Limit: 1, Offset: 1}},
 			want: templateVersions[1:2],
 		},
 		{
 			name: "Last two results",
-			args: args{ctx: ctx, pagination: codersdk.Pagination{Limit: 2, Offset: 8}},
+			args: args{pagination: codersdk.Pagination{Limit: 2, Offset: 8}},
 			want: templateVersions[8:10],
 		},
 		{
 			name: "AfterID returns next two results",
-			args: args{ctx: ctx, pagination: codersdk.Pagination{Limit: 2, AfterID: templateVersions[1].ID}},
+			args: args{pagination: codersdk.Pagination{Limit: 2, AfterID: templateVersions[1].ID}},
 			want: templateVersions[2:4],
 		},
 		{
 			name: "No result after last AfterID",
-			args: args{ctx: ctx, pagination: codersdk.Pagination{Limit: 2, AfterID: templateVersions[9].ID}},
+			args: args{pagination: codersdk.Pagination{Limit: 2, AfterID: templateVersions[9].ID}},
 			want: []codersdk.TemplateVersion{},
 		},
 		{
 			name: "No result after last Offset",
-			args: args{ctx: ctx, pagination: codersdk.Pagination{Limit: 2, Offset: 10}},
+			args: args{pagination: codersdk.Pagination{Limit: 2, Offset: 10}},
 			want: []codersdk.TemplateVersion{},
 		},
 		{
 			name:          "After_id does not exist",
-			args:          args{ctx: ctx, pagination: codersdk.Pagination{AfterID: uuid.New()}},
+			args:          args{pagination: codersdk.Pagination{AfterID: uuid.New()}},
 			expectedError: "does not exist",
 		},
 	}
 	for _, tt := range tests {
 		tt := tt
 		t.Run(tt.name, func(t *testing.T) {
 			t.Parallel()
-			got, err := client.TemplateVersionsByTemplate(tt.args.ctx, codersdk.TemplateVersionsByTemplateRequest{
+
+			ctx, cancel := context.WithTimeout(context.Background(), testutil.WaitLong)
+			defer cancel()
```

Comment:
This could be shorter since the api call shouldn't take that long.

Reply:
I lowered it 👍🏻, didn't initially put any effort into reducing the timeouts. Defaulted everything to high so it would break as little as possible.